content_dev/docs/sdk/java/getting-started/privatehiddenkey.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: codelab
+
+title: PrivateHiddenKey interactions # Step Name
+description: | # SEO Description for this step
+  Keys in Java are different from the Dart SDK AtKeys
```

Comment:
Huh?

Reply:
Have you addressed this?

Reply:
Yes it can be found here
2e6c0fa69d9ae034482ae4e10f4880c6e1d3c9ef